src/VirtualClient/VirtualClient.Actions.UnitTests/FIO/FioMultiThroughputExecutorTests.cs

```diff
@@ -348,6 +348,32 @@ public async Task FioMultiThroughputExecutorCreatesExpectedJobFile_Anomalous_Sce
             }
         }
 
+        [Test]
+        public void FioMultiThroughputFailsIfGroupIDIsNull()
+        {
+            int executions = 0;
+            using (TestFioMultiThroughputExecutor executor = new TestFioMultiThroughputExecutor(this.mockFixture.Dependencies, this.mockFixture.Parameters))
+            {
+                executor.Metadata.Remove("GroupId");
```

Comment:
We can also try metadata with wrong casing

Reply:
Added unit test for bad casing.